plugins/rts54hub/fu-rts54hub-rtd21xx-device.c

```diff
@@ -226,6 +288,14 @@ fu_rts54hub_rtd21xx_device_read_status_cb (FuDevice *device,
 				"status was 0x%02x", status);
 		return FALSE;
 	}
+	else if(status == ISP_STATUS_IDLE_FAILURE)
+	{
+		g_set_error (error,
+			     FWUPD_ERROR,
+			     FWUPD_ERROR_INTERNAL,
+			     "status was 0x%02x", status);
```

Comment:
I think this can be a literal error, you're matching a single value

Reply:
There's no error. Status vaue is ISP_STATUS_BUSY(0xBB) or ISP_STATUS_IDLE_FAILURE(0x12) should return FASLE.

Reply:
But in this block you explicitly match the case `ISP_STATUS_IDLE_FAILURE`, so just use `g_set_error_literal` instead of `g_set_error`